### ISSUE TITLE
Fails to launch on Debian Stretch

### ISSUE THREAD

**Issue by icarito** [juancarlospaco/unicodemoticon#20]
### Bug Short Description:

Short concise description of the problem. No narrative or conversational language.



### Operating System:
*Choose 1 or more. Mark the checkbox it with an `X`.*
- [x] GNU/Linux.
- [ ] Apple Os X.
- [ ] Microsoft Windows.
- [ ] Other or Unknown or Unsupported.



### Bug Severity:
*Choose only 1. Mark the checkbox it with an `X`.*
- [ ] Low, Minor, Aesthetic, Small, Detail, Trivial.
- [ ] Medium, Normal, Standard.
- [ ] High, Important, Major.
- [x] Critical, Catastrophic, Epic Fail.
- [ ] Other or Unknown.



## Version

Current git master head.

### Actual Output:
```
Testing Terminal Colors...
                                                                                 
                                                                                 
                                                                                  

Unix SysLog Server trying to Log to SysLog: /dev/log  🐱 
Logger created with Log file at: /tmp/unicodemoticon.log.  🐱 
Creating Abstract UDP Socket Lock for Single Instance. 
Socket Lock for Single Instance: <socket.socket fd=5, family=AddressFamily.AF_UNIX, type=SocketKind.SOCK_STREAM, proto=0, laddr=b'\x00_unicodemoticon__lock'>. 
Python 3.5.3 on Linux-4.6.0-1-amd64-x86_64-with-debian-9.0.  😹 
STDIN Encoding: UTF-8.  😹 
STDERR Encoding: UTF-8.  🐱 
STDOUT Encoding:UTF-8  🐱 
Default Encoding: utf-8.  😹 
FileSystem Encoding: utf-8. 
PYTHONIOENCODING Encoding: None.  🐱 
Process Name set to: unicodemoticon.  🐱 
Process CPUs Priority set: from 0 to 19. 
using qt5ct plugin
Traceback (most recent call last):
  File "/home/icarito/bin/unicodemoticon", line 7, in <module>
    from unicodemoticon.__main__ import main
  File "/home/icarito/.local/lib/python3.5/site-packages/unicodemoticon/__main__.py", line 59, in <module>
    main()
  File "/home/icarito/.local/lib/python3.5/site-packages/unicodemoticon/__main__.py", line 48, in main
    icon = QIcon(app.style().standardPixmap(QStyle.SP_FileIcon))
TypeError: standardPixmap(self, QStyle.StandardPixmap, QStyleOption, widget: QWidget = None): not enough arguments
ZIP Compressing Unused Old Rotated Logs.  😹 
()  🐱 
```

**Comment by ghost** [comment]
I suspect that `using qt5ct plugin` line...
It does not show up on other distros.

**Comment by ghost** [comment]
WIP Fix for this is on active Development, we hope to have it ASAP, but its happening somewhere else:
https://github.com/juancarlospaco/anglerfish#angler

**Comment by juancarlospaco** [comment]
It should be fixed with recent versions of Qt 5.10+.
